### PR TITLE
Fix single- & multi-node diagrams in App Dev Guide

### PR DESCRIPTION
**Pull request by chenette**
Make diagrams consistent and correct (Sawtooth CLI client is on the validator node).

Includes renaming the Kubernetes multi-node diagram - the previous name included "3TPs", but there are 4 TPs in the example environment.

### CODE REVIEW

**Comment by chenette** [comment]
Reviewers:  The HTML images in the build artifacts appear to have font problems, but that's just a Jenkins build issue. Instead, please see the LaTeX (PDF) images in the build artifacts. 

- Single-node image for Docker / Ubuntu / AWS: https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-1842/2/artifact/docs/build/latex/appdev-environment-one-node-3TPs.pdf

- Single-node image for Kubernetes: 
https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-1842/2/artifact/docs/build/latex/appdev-environment-one-node-3TPs-kube.pdf

- Multi-node image for Docker / AWS: 
https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-1842/2/artifact/docs/build/latex/appdev-environment-multi-node.pdf

- Multi-node image for Ubuntu: 
https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-1842/2/artifact/docs/build/latex/appdev-environment-two-nodes.pdf

- Multi-node image for Kubernetes: 
https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-1842/2/artifact/docs/build/latex/appdev-environment-multi-node-kube.pdf

File: docs/source/app_developers_guide/aws.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ processing.
 .. figure:: ../images/appdev-environment-one-node-3TPs.*
    :width: 100%
    :align: center
-   :alt: AWS application development environment for Sawtooth
+   :alt: AWS: Sawtooth application environment environment with one node
```
Comment (aludvik) [inline docs/source/app_developers_guide/aws.rst]: typo, environment appears twice

File: docs/source/app_developers_guide/docker.rst [hunk 2]
```diff
@@ -31,7 +31,7 @@ processing.
 .. figure:: ../images/appdev-environment-one-node-3TPs.*
    :width: 100%
    :align: center
-   :alt: Docker application environment environment for Sawtooth
+   :alt: Docker: Sawtooth application environment environment with one node
```
Comment (aludvik) [inline docs/source/app_developers_guide/docker.rst]: Typo, environment appears twice

File: docs/source/app_developers_guide/ubuntu.rst [hunk 3]
```diff
@@ -36,7 +36,7 @@ processing.
 .. figure:: ../images/appdev-environment-one-node-3TPs.*
    :width: 100%
    :align: center
-   :alt: Ubuntu application development environment for Ubuntu
+   :alt: Ubuntu: Sawtooth application environment environment with one node
```
Comment (aludvik) [inline docs/source/app_developers_guide/ubuntu.rst]: typo, environment appears twice

**Comment by chenette** [comment]
I pushed a fix-up commit for typos in the alt image text (repeated words) in commit 47ca56ac5..., "Fix alt image text for App Dev guide procedures".

**Comment by peterschwarz** [comment]
Looks like the rename of the images is only partial: There still seems to be a 3TP-named image in there.

**Comment by chenette** [comment]
> Looks like the rename of the images is only partial: There still seems to be a 3TP-named image in there.

This is correct. The remaining image with "3TP" is for the single-node image. Only the multi-node image was renamed.

**Pull request merged**